modules/ROOT/pages/streaming-strategies-reference.adoc

```diff
@@ -3,58 +3,66 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-== Repeatable in Memory Stream Reference
+You can use the default streaming strategy settings or configure then for your
+needs. See xref:streaming-about.adoc[Streaming in Mule Apps] for more details
+on these strategies.
+
+== Repeatable In-Memory Stream Reference
 
 [%header,cols="20,10,10,10,40"]
 |===
 |Parameter Name | Is required? | Expressions | Default Value | Description
-| initialBufferSize
-| NO
-| NO
+| `initialBufferSize`
+| No
+| No
 | 256
-| This is the amount of memory that will be allocated in order to consume the stream and provide random
-access to it. If the stream contains more data than can be fit into this buffer, then it will be expanded
-by according to the bufferSizeIncrement attribute, with an upper limit of maxInMemorySize.
-Default value is 256 KB.
+| Amount of memory allocated to consume the stream and provide random
+access to it. If the stream contains more data than fits into this buffer,
+the memory expands according to the `bufferSizeIncrement` attribute, with an
+upper limit of `maxInMemorySize`.
```

Comment:
The parameter `maxInMemorySize` is presented in two ways accross the doc: 
max**I**nMemorySize and max**i**nMemorySize. 
Parameters are case sensitive, please check which is the correct form and replace all occurrences.

Reply:
there's inMemorySize and maxInMemorySize here. Maybe I'm looking in the wrong file?